core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -197,11 +197,9 @@ public void close() throws SecurityException {
             categories.forEach(new BiConsumer<String, CategoryConfig>() {
                 @Override
                 public void accept(String categoryName, CategoryConfig config) {
-                    final CategoryBuildTimeConfig buildCategory = isSubsetOf(categoryName, buildConfig.categories);
-                    final Level logLevel = getLogLevel(categoryName, config, categories, buildConfig.minLevel);
-                    final Level minLogLevel = buildCategory == null
-                            ? buildConfig.minLevel
-                            : buildCategory.minLevel.getLevel();
```

Comment:
Now that `logBuildTimeConfig` is loaded properly, this would `buildCategory.minLevel.getLevel()` fail due to:
https://github.com/quarkusio/quarkus/blob/2cbd177416f92b9f45abb34b6f01acb8cb0f4eb5/core/runtime/src/main/java/io/quarkus/runtime/logging/InheritableLevel.java#L77
(in case the min-level is _not_ set directly for the given category)

So, I had to use the same approach as for the actual log level (and/or the min-level in LoggingResourceProcessor):